system_tests/gateway_tests.py

```diff
@@ -42,6 +42,9 @@ class TestGateway(BaseTestCase):
     # Firewall Rule
     _firewall_rule_name = 'Rule Name Test'
 
+    # DHCP pool
```

Comment:
My suggestion is to write 'test_0026_add_dhcp_pool' in a new test file.
Refer the 'TestNatRule' which has create, delete and other tests are there.